### PR TITLE
modules/exploits/linux/ssh: Resolve Rubocop violations

### PR DESCRIPTION
**Pull request by bcoles**

### CODE REVIEW

File: modules/exploits/linux/ssh/solarwinds_lem_exec.rb [hunk 1]
```diff
@@ -103,60 +108,61 @@ def exploit
       return
     end
 
-    if ssh
```
Comment (bcoles) [inline modules/exploits/linux/ssh/solarwinds_lem_exec.rb:106]: The entire remainder of this method was wrapped in this `if ssh` conditional with no `else`. I took the opportunity to return early with `return unless ssh` instead.

File: modules/exploits/linux/ssh/symantec_smg_ssh.rb [hunk 2]
```diff
@@ -51,31 +51,23 @@ def initialize(info = {})
...
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => []
+        }
       )
     )
 
-    register_options(
-      [
-        Opt::RHOST(),
```
Comment (bcoles) [inline modules/exploits/linux/ssh/symantec_smg_ssh.rb:60]: Defining `RHOST` was unnecessary.

File: modules/exploits/linux/ssh/symantec_smg_ssh.rb [hunk 3]
```diff
@@ -51,31 +51,23 @@ def initialize(info = {})
...
-
-    register_advanced_options(
-      [
-        OptBool.new('SSH_DEBUG', [ false, 'Enable SSH debugging output (Extreme verbosity!)', false]),
-        OptInt.new('SSH_TIMEOUT', [ false, 'Specify the maximum time to negotiate a SSH session', 30])
-      ]
-    )
-  end
-
-  def rhost
```
Comment (bcoles) [inline modules/exploits/linux/ssh/symantec_smg_ssh.rb:73]: `rhost` is provided by one of the mixins. This helper was unnecessary.

File: modules/exploits/linux/ssh/symantec_smg_ssh.rb [hunk 4]
```diff
@@ -51,31 +51,23 @@ def initialize(info = {})
...
-    datastore['RHOST']
-  end
+    register_options([
+      Opt::RPORT(22)
+    ])
 
-  def rport
```
Comment (bcoles) [inline modules/exploits/linux/ssh/symantec_smg_ssh.rb:77]: `rport` is provided by one of the mixins. This helper was unnecessary.

File: modules/exploits/linux/ssh/solarwinds_lem_exec.rb (hunk 1 above)
Comment (adfoster-r7) [reply modules/exploits/linux/ssh/solarwinds_lem_exec.rb:106]: It looks like it returns early in most scenarios that `ssh` would be set to nil as well 🤔

**Pull request merged**